src/components/tooltip/tooltip.scss

```diff
@@ -0,0 +1,93 @@
+$z-index-tooltip: 2;
+$z-index-tooltip-closer: 5;
+
+.tooltip {
+  color: #00A9EC;
+  margin-left: 10px;
+  position: relative;
+  display: inline;
+  p {
+    margin: 0;
+  }
+  &-questionmark {
+    width: 16px;
+    height: 16px;
+    cursor: pointer;
+    display: inline;
+    &:focus {
+      outline:0;
+    }
+    .icon--questionmark {
+      vertical-align: middle;
+    }
+  }
+  &-popup {
+    position: absolute;
+    top: 24px;
+    left: 50%;
+    transform: translateX(-50%);
+    width: 300px;
+    text-align: left;
+    padding: 13px;
+    background: rgba(0, 0, 0, 0.75);
+    color: rgba(255, 255, 255, 1);
+    z-index: $z-index-tooltip;
+    &__content {
+      &-text {
+        font-size: 11px;
+        p, span, ul {
+          font-size: 11px;
+        }
+      }
+    }
+    &:before {
+      position: absolute;
+      top: -8px;
+      left: 50%;
+      transform: translateX(-50%);
+      display: block;
+      content: '';
+      width: 0;
+      height: 0;
+      border-style: solid;
+      border-width: 0 6px 8px 6px;
+      border-color: transparent transparent rgba(0, 0, 0, 0.75);
+      z-index: $z-index-tooltip;
+    }
+    &__closer {
+      position: fixed;
+      top: 0;
+      right: 0;
+      bottom: 0;
+      left: 0;
+      z-index: $z-index-tooltip-closer;
+    }
+  }
+}
+
+@media only screen and (max-width: 62em) {
```

Comment:
Maybe drop media queries in the lib and then the users can add them themselves so that they can have other breakpoints etc.